src/views/LoginForm/LoginForm.jsx

```diff
@@ -30,12 +30,14 @@ export const LoginForm = ({ handleButtonClick, onChange, hasErrors,alertMessage,
           <div className="hidden__div"></div>
         )}
 
+        <img alt="character logo" src="./billSnapIcon.png" className="character__icon__image"/>
+
         <Form>
           <div className="form__inputs">
             {loginFormInputs.map((inputs, key) => (
               <FormGroup key={key} onChange={onChange}>
                 <FormInput
-                  className="mb-2"
+                  className="register__login__inputs"
```

Comment:
Can you change the LoginForm inputs shape to be more narrow and horizontally long, to match mockup? Similar in shape to what's already in the RegisterForm for the inputs